kalite/main/content_rating_models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+from kalite.facility.models import FacilityUser
+
+from securesync.models import DeferredCountSyncedModel
+
+class ContentRating(DeferredCountSyncedModel):
+    class Meta:
+        unique_together = ("content_kind", "content_id", "user")
+
+    # Maintain info on content type -- could be video, exercise, etc, but we should be able to uniquely id it
+    content_kind = models.CharField(max_length=100, db_index=True, blank=False)
+    content_id = models.CharField(max_length=100, db_index=True, blank=False)
+
+    user = models.ForeignKey(FacilityUser, blank=False)
+    quality = models.IntegerField(blank=False, default=0)
+    difficulty = models.IntegerField(blank=False, default=0)
+    text = models.TextField(blank=True, null=True)
```

Comment:
https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.Field.null

> Avoid using null on string-based fields such as CharField and TextField because empty string values will always be stored as empty strings, not as NULL. If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string. In most cases, it’s redundant to have two possible values for “no data;” the Django convention is to use the empty string, not NULL.